test/plugins/features/movies/index.test.js

```diff
@@ -4,6 +4,8 @@ const Movies = require('../../../../lib/server');
 
 describe('movies integration', () => {
 
+  // payload = { name: 'Volver', release_year: 2020 };
```

Comment:
Why is this comment here?

Reply:
It's some code that I was testing with and forgot to delete after; i'll take it out.